src/confluent_kafka/src/confluent_kafka.c

```diff
@@ -587,6 +593,18 @@ static PyMethodDef Message_methods[] = {
 	  "  :rtype: (int, int)\n"
 	  "\n"
 	},
+	{ "latency", (PyCFunction)Message_latency, METH_NOARGS,
+          "Retrieve the time it took to produce the message, from calling "
+          "produce() to the time the acknowledgement was received from "
```

Comment:
is this more correct? "to the time the delivery report was made available to the application"

Reply:
No, it's the time the ack is received. The time in dr queue is not included